src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -10,6 +10,11 @@ public class Manufacturer {
     public Manufacturer() {
     }
 
+    public Manufacturer(String name, String country) {
+        this.name = name;
+        this.country = country;
+    }
+
```

Comment:
You don't have to create a new constructor without id.
When you create a new object of Manufacturer just set null value to id. 
That'll do nicely. Again it's just a tip, this method is also fine.